### ISSUE TITLE
Tab control is not operable by the keyboard

### ISSUE THREAD

**Issue by Rafael-Luna97** [Rafael-Luna97/github-integration#97]
**Link to finding**  
https://tenanta.localhost-dashboard.essentialaccessibility.com/manual-evaluations/6a6ca99330878d0743e68e3a/results/E1/view?skipScanStandards=true&status=toReview,open,fixed,partiallyFixed,notFixed,notReproducible,cannotBeFixed,validationRequested&linkedPropertyData=6977dc7fbf1fe87270d7645a%7C6977dc7fbf1fe87270d7644d

**Severity**  
Critical

**URL**  
http://google.com

**Actual result**  
The tabs on the  <element>  tab control cannot be navigated using the keyboard only.

**Steps to reproduce**  
1. Go to the URL.
2. Navigate through the page using the keyboard until you reach the tab control.
3. Notice that the tab control   cannot be navigated using the keyboard.

**Issue frequency**  
Individual

**Screen**  
Google

**WCAG Success criteria**  
2.1.1

**WCAG Conformance level**  
A

**Effort**  
High

**Instances**  
None provided

**Disabilities Affected**  
Visual, Physical

**Recommendation**  
The focus should reach the first tab when using the TAB key. The next focusable element using TAB should be the tabpanel, not the other tabs.
Using the LEFT and RIGHT arrow keys should navigate through the different tabs.
ENTER key or SPACE bar key should activate the tab.
1. Add role="tablist" and aria-label attribute to the tabs container.
2. The tabs should be a <button> tag or have role="button" and type="button" properties.
3. Each tab must have the following properties:
       role="tab"
       aria-selected="true/false". The first tab should have this value set to "true".
       aria-controls="ID_TABPANEL"
       id="ID_TAB"
4. While having the aria-selected attribute set to none, the button must have an tabindex attribute set to "-1"
5. For the tabpanel use the following attributes:
      tabindex="0"
       role="tabpanel"
       id="ID_TABPANEL"
       aria-labelledby="ID_TAB"
6. Use JS to add LEFT and RIGHT arrow keys interaction to navigate through the tabs.

 Resources:
 Tab control example:
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html

**Task in the Level Access Platform**  
https://tenanta.localhost-dashboard.essentialaccessibility.com/my-tasks/6a7a30cf410cee05c8b1b588/6a8f4d79596ab29a69adad68

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access
